### PR TITLE
Add 'switchToOutput' setting

### PR DESCRIPTION
**Pull request by voed**
Maybe not best name and description of setting, but you got it i think😊 "Problems" tab more informative and its annoying to switch it manually after each build

### CODE REVIEW

File: src/client/commands.ts [hunk 1]
```diff
@@ -239,4 +248,4 @@ export function compileLocal(outputChannel: VSC.OutputChannel, diagnosticCollect
             doCompile(Path.join(executableDir, potentialFiles[0]), inputPath, compilerSettings, outputChannel, diagnosticCollection);
         });
     });
-}
\ No newline at end of file
+}
```
Comment (WPMGPRoSToTeMa) [inline src/client/commands.ts]: 😞

Comment (voed) [reply src/client/commands.ts]: ¯\\\_(°╭╮°)_/¯

Comment (rsKliPPy) [reply src/client/commands.ts]: It's all good. I've been forgetting newlines at EOFs, will probably add them to all files in one commit.

Comment (WPMGPRoSToTeMa) [reply src/client/commands.ts]: I hate them :D

Comment (rsKliPPy) [reply src/client/commands.ts]: >I hate them :D

https://stackoverflow.com/a/729795 :D

**Comment by voed** [comment]
So, how about merge this? :P

**Comment by rsKliPPy** [comment]
Will do when I come back from vacation. I currently have limited internet access and am spending minimal amount of time on my laptop.

**Comment by voed** [comment]
Oh, have a good rest then🙂

**Comment by voed** [comment]
@rsKliPPy wake up 😒

**Comment by rsKliPPy** [comment]
Woke up!
I do really intend to work on this a bit soon, the exams are almost over. At least to fix some bugs and add finishing touches, but I don't think I will do a complete rewrite of the parser as I think I may move away from Pawn soon.

**Comment by rsKliPPy** [comment]
Thanks! :tada:

**Comment by rsKliPPy** [comment]
Just a note for the next time: this is TypeScript, not JavaScript, so you have to define properties of an object. I've fixed it for you, but take a look at: https://github.com/rsKliPPy/amxxpawn-language/commit/42b700e4ee914c235e0c279cabae1a7e53367e07
Also be consistent with indentations and braces. I know it's up to personal preference, but, as you probably already know, the rule is almost always to stick to whatever is being used throughout the code base when contributing.

**Pull request merged**